docs/index.md

```diff
@@ -0,0 +1,71 @@
+
+---
+title: Home
```

Comment:
I think this is causing a `markdownlint` error because title is not a level 1 heading. You could add `<!-- markdownlint-disable-file MD041 -->` to the top of the page? 
Also might need to add a line before the `title:Home` line for markdownlint too.

Reply:
Sure, I saw it failed the lint but had to add this line to change the title on the sidebar.

Reply:
Removed this line to avoid linting issues. It's not mandatory.